index.js

```diff
@@ -530,6 +541,7 @@ const apiSpec = [
   ['lstat', SF, 'int', ['pointer', 'pointer']],
   ['lstat64', SF, 'int', ['pointer', 'pointer']],
   ['strerror', NF, 'pointer', ['int']],
+  ['unlink', SF, 'int', ['pointer']],
```

Comment:
Would be good to move this right after `readlinkSync` so it follows the order that the functions are defined in.